api/constants/error-messages.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
     MinLength: 'Minimum 3 characters required',
     EmptyField: 'Field can not be empty',
-    FieldNumber: 'Field must be a number',
+    FieldNumber: 'Field must be a positive number',
```

Comment:
I think this should better be called, just to be more descriptive.
```suggestion
    InvalidPriceField: 'Field must be a positive number',
```